src/main/java/frc/robot/RobotContainer.java

```diff
@@ -39,6 +43,8 @@ public RobotContainer() {
 
     // Configure the trigger bindings
     configureBindings();
+
+    new JoystickButton(controllerHID, 9).onTrue(new GrabManipulatorCommand(manipulator));
```

Comment:
Could you stick this in the `configureBindings` method? Just to keep things clean.